packages/paste-core/primitives/tabs/package.json

```diff
@@ -1,7 +1,38 @@
 {
-  "name": "@twilio-paste/primitive-tabs",
-  "version": "0.0.2",
+  "name": "@twilio-paste/tabs-primitive",
+  "version": "0.0.0",
   "category": "interaction",
-  "status": "backlog",
-  "private": true
+  "status": "beta",
+  "description": "An unstyled and accessible basis upon which to style a tabbing system.",
```

Comment:
> tabbing system

Might get confused with focus management. Suggestion could be "a tabset" or "tab list"